webrender_traits/src/types.rs

```diff
@@ -605,9 +605,44 @@ pub enum YuvColorSpace {
 #[derive(Copy, Clone, Debug, Eq, PartialEq, Hash, Serialize, Deserialize)]
 pub struct ExternalImageId(pub u64);
 
+pub trait VectorImageRenderer: Send {
```

Comment:
How about we change this to BlobImageRenderer (and associated types). Blob seems like a reasonable description (Binary Large Object) and makes it clear that this can be used for more than just vector image data.